src/main/java/ru/yandex/practicum/filmorate/controller/FilmController.java

```diff
@@ -0,0 +1,67 @@
+package ru.yandex.practicum.filmorate.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.*;
+import ru.yandex.practicum.filmorate.exception.ValidationException;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import javax.validation.Valid;
+import java.time.LocalDate;
+import java.util.*;
+
+@RestController
+@Slf4j
+@RequestMapping("/films")
+public class FilmController {
+    private int id = 1;
+    private static final LocalDate DATE_BEFORE = LocalDate.of(1895, 12, 28);
+      private final Map<Integer, Film> films = new HashMap();
```

Comment:
Не хватает `<>` после `HashMap`
```suggestion
      private final Map<Integer, Film> films = new HashMap<>();
```
+ лишний пробел, поправь, пожалуйста, класс по code-style: [Ctrl+Alt+L](https://blog.jetbrains.com/idea/2020/03/top-15-intellij-idea-shortcuts/#:~:text=next%20need%20it.-,%E2%8C%A5%E2%8C%98L%20or%20Ctrl%2BAlt%2BL,-We%20can%20easily)

Reply:
Тогда почему тесты пропустили эту ошибку?

Reply:
Это не ошибка, можно оставить без `<>`. Просто мы лишимся дополнительной проверки на нужный тип. Когда мы добавляем тут `<>` - компилятор автоматически проверяет, что в качестве ключа у нас должен быть только объект класса `Integer`, а в качестве значения  - `Film`, просто для сокращения кода дублирование параметров опускается и остается только `<>`

Reply:
Понятно

_Сообщения доставляются моментально из этой ветки, без каких-либо дополнительных манипуляций? Как чат/ мессенджер?_

Reply:
Мне приходит отбивка на почту, что сделано изменение :)

Но все же в качестве мессенджера лучше использовать слак. А комментарии к pr - для подсвечивания каких-либо моментов в коде